bin/melc.ml

```diff
@@ -10,6 +10,7 @@
 (*                                                                     *)
 (***********************************************************************)
 
+open Melange_compiler_libs
```

Comment:
Why was this needed?

Reply:
otherwise it doesn't know about coming from `Melange_compiler_libs.Unit_info`.